Cc backend setup

OpaJaa opened the pull request:
Skeleton server for the backend. Starts with "npm start" and uses port 3000.

rvsape commented:
add .gitignore and ignore node_modules/

OpaJaa commented:
> add .gitignore and ignore node_modules/

Done. Used the one mentioned by Ali in his pull request. Didn't notice we lacked one and just committed and pushed all... Sorry! :D

bunjovis commented:
As @rvsape commented in Teams chat, Typescript support should also be added already

OpaJaa commented:
> As @rvsape commented in Teams chat, Typescript support should also be added already

ts support added using the tutorial @rvsape linked to chat.

bunjovis commented:
I think this is still missing an example dotenv file (.env.example for example), which contains all the necessary env configuration parameters with no secrets. Also something in readme about creating the .env file before it can be run.

OpaJaa commented:
> I think this is still missing an example dotenv file (.env.example for example), which contains all the necessary env configuration parameters with no secrets. Also something in readme about creating the .env file before it can be run.

Added .env.example. It only has the port for now. Also added better instructions to readme. Let me know, if the instructions have something wrong in them or need something more added.

rvsape commented on cc-backend/readme.md:
```diff
@@ -0,0 +1,7 @@
+- Clone from repo.
```
some mention of the technologies used, like "Express Typescript server"
formatting guide: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

OpaJaa replied:
Updated readme with some technologies and versions. Should discuss together what and add there and how we format them so every readme would look similar.

rvsape commented:
need to come up with some consensus about the ts - configurations in every service but its a problem for another PR
